.github/workflows/qiita-ci.yml

```diff
@@ -66,6 +67,7 @@ jobs:
           conda config --add channels conda-forge
           conda create -q --yes -n qiita python=3.9 libgfortran numpy nginx cython redis
           conda activate qiita
+          python -m pip install --upgrade pip
```

Comment:
Do we really need this?

Reply:
I added them as part of updating the qiita installation, but I can certainly remove them and confirm that it still works as intended.